src/MetricActionBar.tsx

```diff
@@ -86,6 +100,73 @@ export class MetricActionBar extends SceneObjectBase<MetricActionBarState> {
     });
   };
 
+  public getPanelData = (): PanelDataRequestPayload => {
```

Comment:
```suggestion
  public getPanelData = (vizPanel): PanelDataRequestPayload => {
```

By modifying this function to accept a panel as argument, we make it easier to reuse in the future, in a different context (e.g. if we add a menu item on the panel itself).